_posts/2017-11-30-foreman-community-newsletter---november-2017.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Foreman Community Newsletter - November 2017
+date: 2017-11-30 12:59:00
+author: Ori Rabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Migrating from mailing lists, Katello 3.5 RC, and the usual round-up
+---
+
+Important news for mailing list users! Also releases, demos, and all
+that goodness!
+
+### Migrating from Google Groups
+
+Last month Greg made a proposal to migrate away from the Google Groups
+mailing list to a dedicated Foreman community forum. You can read more
+about his reasoning [here][discourse_users], or watch the
+[AMA](https://www.youtube.com/watch?v=IdV6US_d4-U).
+
+The debate is not closed yet, so please do join in, either on the list
+thread above, or on the [migration poll][migration_poll] if you have views
+on this. At this time, the likely outcome is that we *will* move to
+Discourse in the near future, so watch out for more announcements about
+that in the coming weeks.
+
+### Katello 3.5 RC2
```

Comment:
I think that @akofink is working on the final release. Also note that @dLobatog is working on building the foreman 1.16.0 release. might be good to consider that too.

Reply:
Yes, it's quite impossible (afaik) to build plugin releases (i.e. Katello) without having built Foreman first; however, out goal is to release 3.5 GA as quickly as possible after Foreman's GA.